examples/std/.cargo/config.toml

```diff
@@ -0,0 +1,4 @@
+[build]
+target = "x86_64-unknown-linux-gnu"
+[target."x86_64-unknown-linux-gnu"]
```

Comment:
Same here, I'm not aware of a way to override the runner for the host triple. So this needs adapting for e.g., OSX.

Reply:
Same here, I'll add a comment.